secretless-broker.rb

```diff
@@ -2,16 +2,16 @@
 class SecretlessBroker < Formula
   desc "Secures your apps by making them Secretless"
   homepage "https://secretless.io"
-  version "1.5.0"
+  version "1.4.2"
```

Comment:
This seems like an error or a bad push

Reply:
Oh that's weird. Was there a new pr merged while mine was pending? I'll fix that